.travis/travis-openmandriva.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd $SCRIPT_DIR
+
+source $SCRIPT_DIR/travis-common.inc
+
+set -e
+set -x
+
+JOB_NAME=${TRAVIS_JOB_NAME:-OpenMandriva cooker}
+
+arr=($JOB_NAME)
+release=${arr[1]:-cooker}
+
+mmd_run_docker_tests \
+    os=openmandriva \
+    release=$release \
+    repository=hub.docker.com/r \
```

Comment:
This address gives a 404. Are you sure you don't mean
```suggestion
    repository=docker.io \
```

Because that pulls an image successfully.

Reply:
There is something strange here as OpenMandriva docker's images lives on hub.docker.com/r/openmandriva previously I've set docker.io and it did not work, while docker pull openmandriva/cooker works